onboarding.qmd

```diff
@@ -16,6 +16,40 @@ We value believe in investing in and nurturing scientists as much as the science
 
 *Links and details to follow.*
 
+# Roles and expectations:
+
+## Undergraduates
+-   
+-   
+-   
+
+
+## Research associates (PRAs)
+-   
+-   
+-   
+
+
+## Graduate students
+-   
+-   
+-   
+
+
+## Postdoctoral fellows (Postdocs)
+
+As a postdoctoral researcher, you wear many hats in addition to those of more junior scientists in the lab.
+
+-   A postdoctoral position is a training opportunity -- you are here to learn new skills and expand your scientific boundaries. You are not expected to know everything, but you should be comfortable (and preferably excited!) throwing yourself into new problems and solving them.
```

Comment:
Maybe one-word/phrase subheadings?
- _Training_: ...
- _Mentoring & Zen_: 
- _Collaboration_: along the lines of ... Definitely lead a few projects + collaborate with others within and outside the group. Line 46 (independent projects before other stuff in Line 45)? 
- _Troubleshooting_:
- _Funding_: _Just keep writing!_
- _Communication_:
- _Find yourself_: